cdap-common/src/main/java/co/cask/cdap/common/lang/HadoopClassExcluder.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.common.lang;
```

Comment:
Doesn't feel like it goes to lang (lang is for core Java language features, like Class, Type, ClassLoader that kind of stuff. Maybe better put it under co.cask.cdap.common.twill for now.